pupil-spa/src/app/login-success/login-success.component.html

```diff
@@ -4,17 +4,17 @@
         <div class="grid-row">
             <div class="column-full">
                 <header class="page-header">
-                    <h1 class="aa-title-size" tabindex="0">Welcome, {{ pupil.firstName }}</h1>
+                    <h1 class="aa-title-size" tabindex="0">Welcome, {{ pupil.firstNameAlias || pupil.firstName }}</h1>
```

Comment:
Why store two properties in the SPA?  Why not just call it `firstName`?  It is always the pupil view of their name.

Reply:
as long as this is implemented in the payload generation code, and not the underlying view, i agree

Reply:
as long as this is implemented in the payload generation code, and not the underlying view, i agree

Reply:
This is now reverted to display `firstName` and `lastName` since it's processed now prior to message queue submission.